src/core/dfn-index.js

```diff
@@ -0,0 +1,214 @@
+// @ts-check
+/**
+ * If a `<section id="index">` exists, it is filled by a list terms defined by
+ * reference (external terms).
+ */
+
+import { addId } from "./utils.js";
+import { citeDetailsConverter } from "./data-cite.js";
+import { fetchAsset } from "./text-loader.js";
+import { getTermFromElement } from "./xref.js";
+import { hyperHTML as html } from "./import-maps.js";
+import { renderInlineCitation } from "./render-biblio.js";
+import { sub } from "./pubsubhub.js";
+
+export const name = "core/dfn-index";
+
+/**
+ * @typedef {{ term: string, type: string, linkFor: string, elem: HTMLAnchorElement }} Entry
+ */
+
+export async function run(conf) {
+  const index = document.querySelector("section#index");
+  if (!index) {
+    return;
+  }
+
+  const styleEl = document.createElement("style");
+  styleEl.textContent = await loadStyle();
+  document.head.appendChild(styleEl);
+
+  index.classList.add("appendix");
+  if (!index.querySelector("h2")) {
+    index.prepend(html`<h2>Index</h2>`);
+  }
+
+  const toCiteDetails = citeDetailsConverter(conf);
+
+  const externalTermIndex = html`<section id="index-defined-elsewhere">
+    <h3>Terms defined by reference</h3>
```

Comment:
as above... I guess all the headings are localization targets.

Reply:
How about `[SPEC] defines the following: `?